src/index.ts

```diff
@@ -1,14 +1,20 @@
 require('dotenv').config()
 
+import * as Sentry from '@sentry/node'
 import { PORT, URL_PREFIX } from './config'
 import express from 'express'
 import expressPlayground from 'graphql-playground-middleware-express'
 import cors from 'cors'
 import GraphQLMiddleware from './graphql'
 import TypeAheadMiddleWare from './typeahead'
 
+Sentry.init({ dsn: 'https://faaffe3ccc10486492cae5436114c4b5@sentry.data.amsterdam.nl/52' })
```

Comment:
FYI I noticed that we're actually are getting a DSN from the environment (see the openstack config) we might want to use that.

Reply:
Ah, nice one. Changed it.